misc/snmp.conf

```diff
@@ -0,0 +1 @@
+mibdirs +/usr/share/snmp/mibs +/opt/librenms/mibs +/opt/librenms/mibuploader/snmpttmibs
```

Comment:
I'm not sure but /usr/share/snmp/mibs can probably cause issues so I'd remove that

Reply:
Why this path causes a problem ?